package.json

```
@@ -110,7 +110,7 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.1",
-    "flow-bin": "^0.72.0",
+    "flow-bin": "^0.89.0",
```

let's first update to latest `^0.79.1` 
```suggestion
    "flow-bin": "^0.79.1",
```

```basg
⇒  npm run test:flow      

> redux-form@8.2.4 test:flow /home/aluca/Projects/redux-form
> node scripts/patch-immutable-flow-error.js && opencollective-postinstall && flow check

Thank you for using redux-form!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/redux-form/donate

Found 0 errors
```

---

also please rebase